source/documentation/device_wifi.md

```diff
@@ -0,0 +1,28 @@
+# Certificate Authentication (Alpha)
+
+## What is Certificate Authentication
+Device-Wifi requires a user to have either a government email address, be sponsored by someone with a government email address, or have a mobile phone number. Device-Wifi only requires that a certificate needs to be installed on a device to gain access, this removes the need to always enter an email and password to gain access to government Wifi.
+
+## How it works
+1. Register your organisation with the GovWifi Admin portal and setup the “GovWifi” SSID on your local network.
+2. Setup a “Certificate Authority” using a Public Key Infrastructure (PKI) software tool. This allows the organisation to sign certificates.
+3. Using your PKI, sign client certificates.
+4. Provide us with your root CA certificate used to sign these client certificates.
+5. Devices can now connect to the internet via GovWifi wherever GovWifi is offered.
+
+![logo]
+
+## Where is the code
+You can find the code for Certificate Authentication within the [govwifi-frontend](https://github.com/alphagov/govwifi-frontend) repo
```

Comment:
We need two things here:
1. The exact file where EAP-TLS is handled in FreeRADIUS
2. The folder where the root CA certificates are held.